tooling/cli/lib/deploy.js

```diff
@@ -196,31 +195,43 @@ async function triggerBatch(instance, key) {
 }
 
 async function getHeliumUploadData(instance) {
-  const endpoint = instanceEndpoint(instance);
-
-  const options = {
-    method: 'POST',
-    headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify({ query: KEY_QUERY, variables: { nickname: instance.nickname } })
-  };
-
-  let responseData = {};
-  const launchData = await fetch(endpoint, options).then(r => r.json());
-
-  if (launchData && launchData.data) {
-    const {
-      data: {
-        HeliumLaunchData: { key, signedUrl }
-      }
-    } = launchData;
+  return new Promise((resolve, reject) => {
+    const endpoint = instanceEndpoint(instance);
 
-    responseData = {
-      key,
-      signedUrl
+    const options = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ query: KEY_QUERY, variables: { nickname: instance.nickname } })
     };
-  }
 
-  return responseData;
+    let responseData = {};
```

Comment:
Can just move this into the `if` block on L212 and define it as `const` at L220.